test/integration/query-interface/createTable.test.js

```diff
@@ -75,16 +74,19 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
           expect(indexes[1].unique).to.be.true;
           expect(indexes[1].fields[0].attribute).to.equal('name');
           break;
+
         case 'mariadb':
         case 'mysql':
-        // name + email
+          break;
```

Comment:
isn't this break preventing the code below to be executed for Maria and mysql?

Reply:
Again not sure if I did it as it is not specific to db2 but reverting.